content/doc/addons/heptapod.md

```diff
@@ -0,0 +1,229 @@
+
+The type of the instance is "Heptapod Docker Runner", and is identified by a Heptapod icon. The job number is included of the Application name, which looks like hpd-job-kvzWdEMo-275833.
+
+Both Application and Instance are removed as soon as the job completes, whatever end status was returned to the coordinator. You will be billed for the time the instance was up, with a 10-second granularity.
+
+The pricing will vary on the selected flavour (size, see also below), which is "M" by default. Check it out on the Clever Cloud pricing page, under the "Heptapod Runner" section.
+
+#### Activation
+
+On <heptapod.host>, the Shared Runners are disabled by default at Group creation time, and Projects created while they are disabled on the Group also will have them disabled.
+
+This is nothing but the standard GitLab mechanism. If you maintain your own Group or Project runners, this makes sure the Clever Cloud Runner does not steal jobs from them, as chances are high that they would require a specific setup. Also, this avoids billing users for a service that they did not ask for and may not be aware of.
+
+In order to have the Clever Cloud Runner take your jobs, simply activate the shared runners on the wished Groups and Projects:
+
+* Navigate to the Group Settings CI/CD page.
+* Expand the Runners section.
+* You will be able to activate the shared runners directly from there or allow Projects or sub-Groups to override the Group setting in case you don't want to activate on the full Group.
+* Always check the resulting configuration in the Projects Settings CI/CD pages, and correct it if needed. Changes in the Group Settings are not always immediately taken into account on enclosed Projects.
+
+#### Running only the Clever Cloud Runner
+
+In case you'd like to make sure that your jobs are taken by the Clever Cloud Runner, only, you can flag them with the `clever-cloud` job tag.
+
+Here is an example where the tag is set on all the jobs in the CI/CD configuration in one stroke, thanks to the `default` global keyword:
+
+```yaml
+default:
+  tags:
+    - clever-cloud
+
+lint:
+  script:
+    - ci/lint
+
+tests:
+  script:
+    - ci/run-all-tests
+```
+
+#### Working like any Docker-based Heptapod or GitLab Runner
+
+Except for provisioning, everything happens as if the job was launched by a standard Docker executor, with only a handful of differences (see the dedicated subsection).
+
+{{< callout type="warning" >}}
+  For Git projects, there is no difference between Heptapod Runner and GitLab Runner
+{{< /callout >}}
+
+So typically, you would create a Docker image with all the needed dependencies, perhaps host it on registry.heptapod.host, and use it as the base context for your job.
+
+Notably, the job can make use of services – these will run inside the same virtual machine as the main job.
```

Comment:
"The job can make use of services, which will run inside the same virtual machine as the main job".